src/CrandallGridWorld/TeamBug.java

```diff
@@ -1,62 +1,59 @@
-package CrandallGridWorld;
+package ourTeam;
 
+import java.lang.reflect.Constructor;
 import java.util.ArrayList;
 
 import info.gridworld.actor.Actor;
 import info.gridworld.actor.Critter;
 import info.gridworld.grid.Location;
 
-//Version 1.1.0
+//Version 1.11
 //@Author Suchir/Billy/Tim, and everyone who helped figure this out on the whiteboard
 //Any edits/improvements are appreciated
-//Not sure if we need final for the methods, but it seemed useful.
-
+//Not sure if we need final for the methods, but it seemed useful
 public abstract class TeamBug extends Critter {
 	private int team;
-	private boolean isSterile;
 	private int turnsToDeath;
+	private int turnsToUnfreeze;

```

Comment:
This should not be all in one commit. It is too confusing knowing what is just formatting and what is functional changes.